pkg/monitors/vsphere/model/model.go

```diff
@@ -46,6 +56,13 @@ type Config struct {
 	// See https://github.com/antonmedv/expr for more advanced syntax.
 	Filter string `yaml:"filter"`
 
+	// The host dimension value set for monitored VMs.
+	// The options are: `ip`, `hostname` and `disable`. Default is `ip`.
+	// `ip`   : the VM IP if available
+	// `hostname` : the VM Hostname if available
+	// `disable`: the vsphere monitor does not set the host dimension on the VM metrics
+	VMHostDimension VMHostDim `yaml:"vmHostDimension" validate:"required" default:"ip"`
```

Comment:
I suppose this will have to be changed if we want to make this optional (don't want to break existing configs).